src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -147,10 +205,14 @@ public function getAccessibleWorkspaceNames()
      */
     public function getNode($path)
     {
+        if (empty($this->caches['nodes'])) {
+            return parent::getNode($path);
```

Comment:
i think i understand now. but anyway, i will wait for #280 to be merged and then we can rebase this one. will make it easier to review.